node/src/actors/chain_manager/mining.rs

```diff
@@ -129,8 +130,9 @@ impl ChainManager {
         let is_ars_member = rep_engine.is_ars_member(&own_pkh);
 
         let superblock_period = u32::from(chain_info.consensus_constants.superblock_period);
-        // FIXME: Only ARS members from the last block of the previous SuperBlock can create a SuperBlock
-        if rep_engine.is_ars_member(&own_pkh) && current_epoch % superblock_period == 0 {
+        // Everyone creates superblocks, but only ARS members broadcast it
+        // TODO: add config flag to disable superblock creation?
+        if current_epoch % superblock_period == 0 {
             // FIXME(#1236): ARS Members have to include the BLS signature instead of PublicKeyHash
             // FIXME: After Reputation Merkelitation, only the ARS Members from the previous consolidated
             // block will be added, to avoid include addresses that could be wrong later by
```

Comment:
```suggestion
            // Superblock will be added, to avoid include addresses that could be wrong later by
```

Reply:
I am not sure how the ARS members is saved. Because we need the last ARS Members, that are the only nodes allowed to bootsrap the SuperBlockVotes, but you also need the ARS member from the previous consolidated block that will be the future nodes allowed to send the SuperBlockVotes

Reply:
I was thinking that when calling `build_superblock` number 10 we will pass the list of ARS identities from epoch 98 (or 99?). That list can be saved and used to validate all the superblock votes with index = 10. In order to validate superblocks with index = 11 we must wait to create the superblock number 11 locally. And in order to validate superblocks with index = 9, we could store a copy of the old list of members, but this PR just forwards it (after validating the secp256k1 signature). We should assess whether to save old superblock votes is useful, then we can implement validation of old votes.

Reply:
Okay, only ARS members from epoch 88 can create the superblock number 10. So we need to store two copies: ARS members from this superblock and ARS members from the previous superblock.